### PR TITLE
Do not use signature deprecated since SQLAlchemy 0.9.

### LINKED ISSUE #54: DeprecationWarnings with SQLAlchemy 1.4.0b1
In `SQLAlchemy 1.4` a some new `DeprecationWarning`s were issued. 

Most prominent, the old-style method signature of after_bulk_update and after_bulk_delete changed to a take a `[update|delete]_context` similar to `after_flush` https://github.com/zopefoundation/zope.sqlalchemy/blob/master/src/zope/sqlalchemy/datamanager.py#L280-L284

This style was introduced in version 0.9 and would also require to bump the minimal required version of SQLAlchemy to this version. https://docs.sqlalchemy.org/en/14/orm/events.html?highlight=after_bulk_update#sqlalchemy.orm.SessionEvents.after_bulk_delete

### PR DESCRIPTION
**Pull request by sallner**
This does not introduce complete compatibility of this package with SQLAlchemy but avoids some deprecation warnings. 

Fixes #54.

### CODE REVIEW

**Review by mgedmin — COMMENTED** [review]
LGTM, but I think maybe we should set up GitHub Actions here so we have working CI.

https://github.com/zopefoundation/meta/ has templates and tools for doing this, although the tools could be better documented.

**Comment by sallner** [comment]
> LGTM, but I think maybe we should set up GitHub Actions here so we have working CI.
> 
> https://github.com/zopefoundation/meta/ has templates and tools for doing this, although the tools could be better documented.

Oh, I did not realize, that the credits are already out. Will do that soon.

File: CHANGES.rst [hunk 1]
```diff
@@ -8,6 +8,9 @@ Changes
   ``ZopeTransactionEvents``.
   (`#46 <https://github.com/zopefoundation/zope.sqlalchemy/issues/46>`_)
 
+- Reduce ``DeprecationWarning``s with SQLAlchemy 1.4.
```
Comment (mgedmin) [inline CHANGES.rst:11]: GitHub's ReStructuredText renderer doesn't like this.

restview confirms that this is a problem:

> System Message: WARNING/2 (CHANGES.rst, line 11); backlink
>
> Inline literal start-string without end-string.

```suggestion
- Reduce DeprecationWarnings with SQLAlchemy 1.4.
```

**Review by mgedmin — APPROVED** [review]
One ReST problem; the rest continues to LGTM.

File: setup.py [hunk 2]
```diff
@@ -50,7 +50,7 @@
 
     install_requires=[
         'setuptools',
-        'SQLAlchemy>=0.7',
+        'SQLAlchemy>=0.9',
```
Comment (icemac) [inline setup.py:53]: This change should be mentioned in the change log.

**Review by icemac — CHANGES_REQUESTED** [review]
LGTM besides one feature request.

**Review by mgedmin — APPROVED** [review]

**Review by icemac — APPROVED** [review]

**Pull request merged**